src/index.js

```diff
@@ -1,7 +1,23 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './App';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import App from './components/App';
 import * as serviceWorker from './serviceWorker';
+import rootReducer from './reducers/index';
+import INITIAL_STATE from './constants/InitialState';
+
+const STATE = { books: INITIAL_STATE };
+
+const store = createStore(rootReducer, STATE);
+
+ReactDOM.render(
+  <Provider store={store}>
+    <App />
+  </Provider>,
+  document.getElementById('root'),
+);
+serviceWorker.register();
 
 ReactDOM.render(
```

Comment:
**REQUIRED**
* Rendering twice is throwing an error since the last render is executed last and does not have access to the store.
Let's delete it.